From ce04fe5f8eb9f3a27504db75672083c8aaf80ddd Mon Sep 17 00:00:00 2001
From: Aaron Ballman <aaron@aaronballman.com>
Date: Tue, 19 Jul 2016 17:46:55 +0000
Subject: [PATCH] This code block breaks the docs build
 (http://lab.llvm.org:8011/builders/llvm-sphinx-docs/builds/11920/steps/docs-llvm-html/logs/stdio),
 but I cannot see anything immediately wrong with it and cannot reproduce the
 diagnostic locally. Setting the code highlighting to none instead of nasm to
 hopefully get the bot stumbling back towards green.

git-svn-id: https://llvm.org/svn/llvm-project/llvm/trunk@275998 91177308-0d34-0410-b5e6-96231b3b80d8
---
 docs/AMDGPUUsage.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/AMDGPUUsage.rst b/docs/AMDGPUUsage.rst
index 34a9b60..7d1ef11 100644
--- a/docs/AMDGPUUsage.rst
+++ b/docs/AMDGPUUsage.rst
@@ -171,7 +171,7 @@ keys, see the comments in lib/Target/AMDGPU/AmdKernelCodeT.h
 
 Here is an example of a minimal amd_kernel_code_t specification:
 
-.. code-block:: nasm
+.. code-block:: none
 
    .hsa_code_object_version 1,0
    .hsa_code_object_isa
-- 
2.5.5

